### ISSUE TITLE
Got "Exception occurred! 'verifY'" when try to remove apb from ansible-service-broker

### ISSUE THREAD

**Issue by ghost** [fusor/ansible-playbook-bundle#81]
```
➜  my_apb git:(master) ✗ apb remove --all --broker http://localhost:1338
Exception occurred! 'verifY'
```

**Comment by wjiangjay** [comment]
[fix code](https://github.com/fusor/ansible-playbook-bundle/pull/82)

**Comment by ghost** [comment]
fix code: https://github.com/fusor/ansible-playbook-bundle/pull/82

**Comment by jmrodri** [comment]
Fixed by PR #82
